prowler/lib/mutelist/mutelist.py

```diff
@@ -119,7 +118,7 @@ def __init__(
             self._mutelist = mutelist_content
 
         if self._mutelist:
-            self.validate_mutelist()
+            self._mutelist = self.validate_mutelist(self._mutelist)[1]
```

Comment:
This way is more Pythonic and we clarify that is a static utility 
```suggestion
            self._mutelist = Mutelist.validate_mutelist(self._mutelist)[1]
```

Reply:
Done ✅